RESULTS.md

```diff
@@ -570,5 +590,94 @@ examples from the first 400 moves should mostly rid us of those pointless
+used the BigTable input readers to enforce a 90:10 ratio.
+
+Another benefit of the BigTable integration was that we could directly run a
+query on BigTable to compute the appropriate resign threshold. Previously, this
+had been done by manually inspecting a Stackdriver graph and editing a flagfile
+on GCS. This meant that the latency of resign threshold updates dropped from
+hours to minutes.
+
+Overall, time between checkpoints dropped from 30 minutes to 20 minutes. (With
+one run being 800 checkpoints, this means that we can complete a run in 11 days.)
+We had much more consistent sampling for resign disabled/enabled games and for
+games of varying lengths. We reduced the latency for resign threshold measurements.
+
+The run was uneventful and was a bit better than previous runs; it escaped random
+play much more quickly, probably because the improved resign threshold latency
+meant that our training data was appropriately pruned of 'easy' positions.
+So all in all, the rewrite didn't create any new pipeline issues.
+
+Around generation 280, we discovered on LCZero forums that AlphaGoZero had used
+init-to-loss for Q values. We switched to init-to-loss from init-to-parent and
+saw our network gain dramatically in strength. Soon, our top model thoroughly
+exceeded our previous best model, and beat LZ's v157, the last model from before
+LZ started mixing in game data from the much stronger ELF.
+
+### v15, clean run with init to loss
+
+This run beat v14 pretty early on, producing a model that was stronger than v14's
+best at generation 330. v15 ended up slightly stronger than v14 but was otherwise
+similar.
+
+Qualitatively, we noticed that V14/V15's models were much sharper,
+meaning that they had more concentrated policy network output and the value
+network output would have stronger opinions on whether it was winning or losing.
+Simultaneously, the calibrated resign threshold for init-to-loss ended up at 0.5
+compared to 0.9 in previous runs. So v14/v15's MCTS was operating in the sweet
+spot more frequently. (The MCTS algorithm tends to go blind once Q gets close to
+0.9, because there's no more room at the top.) The init-to-loss configuration
+also led to much deeper reads, as the tree search behavior would dive deeply
+into one variation before going onto the next. This read behavior meant that
+v14 was our first model to be able to consistently read ladders. As a result,
+it was actually not clear whether v14 was actually stronger than v12, or whether
+v14 was just bullying v12 around by reading ladders that v12 couldn't read, and
+winning easy rating points. But since we had achieved success against external
+reference points, we concluded that v14/v15 were in fact really stronger than v12.
+
+During v15's run, we realized that our evaluation cluster infrastructure was in
+need of some upgrades, primarily for two reasons:
+
+1. We wanted to include LZ in the mix, so we could cross-evaluate LZ models with
```

Comment:
i'd say we want to evaluate against models with a different "genealogy"; it's not LZ specific.